thirdeye/thirdeye-frontend/app/pods/components/rootcause-select-metric-dimension/template.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="col-xs-4">
```

Comment:
Perhaps we could move the `.row` div wrapper inside the component (https://github.com/linkedin/pinot/pull/2234/files#diff-1bf4da1ba2e97196b1a3e10e3c1ed6ddR80) , since boostrap's col-*-* must be wrapped in one. (it's easier than having to remember to add them in each template where the component is used.

Reply:
yeah it not currently uniform and we'll have to refactor a bunch of these.